src/frontend/qt_sdl/Input.h

```diff
@@ -26,6 +26,7 @@ namespace Input
 
 extern int JoystickID;
 extern SDL_Joystick* Joystick;
+extern SDL_Haptic *Haptic;
```

Comment:
Could you change this as well to keep with the current coding style?
```suggestion
extern SDL_Haptic* Haptic;
```